frontend/src/pages/modelServing/screens/projects/nimUtils.ts

```diff
@@ -46,24 +45,22 @@ export const getNIMData = async (isNGC: boolean): Promise<Record<string, string>
   return data;
 };
 
-export const isNIMSupported = (currentProject: ProjectKind): boolean => {
+export const isProjectNIMSupported = (currentProject: ProjectKind): boolean => {
   const isModelMeshDisabled = currentProject.metadata.labels?.['modelmesh-enabled'] === 'false';
   const hasNIMSupportAnnotation =
     currentProject.metadata.annotations?.['opendatahub.io/nim-support'] === 'true';
 
   return isModelMeshDisabled && hasNIMSupportAnnotation;
 };
 
-export const isNIMAPIKeyEnabled = async (dashboardNamespace: string): Promise<boolean> => {
-  try {
-    const configMap = await getConfigMap(dashboardNamespace, NIM_API_KEY_VALIDATION);
+export const isNIMServingRuntimeTemplateAvailable = async (): Promise<boolean> => {
+  const TEMPLATE_NAME = 'nvidia-nim-serving-template';
+  const NAMESPACE = 'redhat-ods-applications';
```

Comment:
Can't do this -- you need to get access to the `useNamespaces` -- never hardcode any namespace in our app. We are heavily configurable.

Reply:
Updated